recipes/panflute/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Sergio Correia
```

Comment:
Please make an issue/PR upstream to make sure the license is included in the pypi tarball.

Reply:
but why?

Reply:
Maintenance and the license probably requires it for the pypi dist. Not a big deal and this recipe is merged.

Reply:
Once it is in the pypi tarballs, then you simply put the name of the file. Conda build will find it.